metadata/schemas/sensor.json

```diff
@@ -51,32 +51,22 @@
                 "type": {
```

Comment:
sorry for the confusion. The metadata schema are to be installed when we use the package (e.g. using pip or conda) such that others could use that when doing pip install digitaltwin. As such, we would have to move that in the directory digitaltwin, so please move that to digitaltwin/metadata/schemas/sensor.json. Sorry for the confusion.
The test directory should also have the same structure as digitaltwin, (e.g. test/metadata/schemas/sensor.json) to test exactly those entities defined in the src. That makes it much easier to find the tests to be modified and vs versa see what the problem is in case of failed tests. So it might be helpful to rename validate_metadata/../test_sensor.py accordingly.

Reply:
I have reorganized this.